app/controllers/host/events_controller.rb

```diff
@@ -40,7 +40,10 @@ def new
     @event = Event.new
   end
 
-  def edit; end
+  def edit
+    @event = current_host.events.find_by(id: params[:id])
```

Comment:
jeśli `find_by(id: params[:id])` zastąpimy `find(params[:id])` 
to linijka
`raise ActiveRecord::RecordNotFound if @event.nil?`
nie będzie potrzebna
`.find()` rzuca wyjątkiem jeśli nie znajdzie rekordu o podanym ID